utils/cargo-tfhe-lints-inner/rust-toolchain.toml

```diff
@@ -0,0 +1,4 @@
+[toolchain]
+channel = "nightly-2024-05-02"
+components = ["rustc-dev", "rust-src", "llvm-tools-preview"]
+profile = "minimal"
```

Comment:
could this file (or the other toolchain file) be a symlink to have the toolchain file defined once for the inner tool and the wrapper ?

Reply:
this other one should be removed actually

Reply:
done